docs/spark-standalone.md

```diff
@@ -264,6 +266,8 @@ SPARK_MASTER_OPTS supports the following system properties:
   <td>
     Path to resources file which is used to find various resources while worker starting up.
     The content of resources file should be formatted like the <code>ResourceAllocation</code> class.
+    Unlike discovery script, resources file should always assign different resource addresses for
+    workers/drivers(client only) whether <code>spark.resources.coordinate.enable</code> is off or not.
```

Comment:
I didn't think we were going to make this true.  I would prefer discovery script and resourceFile parameter to have the same behavior and the config controls which it does. I need to finish looking at the updated code

Reply:
> I would prefer discovery script and resourceFile parameter to have the same behavior and the config controls which it does.

I was think that our resources file should always assign different resources for workers/drivers. If resources file could have the same behavior with discovery script, it would be more friendly for users.

Reply:
yeah I checked the code and we aren't differentiating resourcesFile vs discovery script, lets just remove that line from comment.